ihp.cabal

```diff
@@ -187,12 +186,13 @@ library
         , IHP.Controller.Layout
         , IHP.Controller.BasicAuth
         , IHP.Controller.Cookie
+        , IHP.Controller.AccessDenied
+        , IHP.Controller.NotFound
```

Comment:
```suggestion
        , IHP.Controller.NotFound
        , IHP.Controller.Response
```

Reply:
@amitaibu  `IHP.Controller.Response` still needs to be added here to fix the build